repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,132 @@
+#########################
+# Gatling Configuration #
```

Comment:
I should note that this file is a copy of the default configuration file installed with Gatling. The only change was adding and configuring a Graphite writer.